### PR TITLE
Fix #765 from XSK, Fixes #1676, #1677

### PR DESCRIPTION
**Pull request by dimitarquanterall**
Add New -> xsaccess file in workspace contextual menu. 
If file already exists in destination folder the option is not displayed.  

[Fixes also #1676, #1677](https://github.com/dimitarquanterall/ide-workspace/commit/b8f57fa7b9172448fdc13943d53655e4e000e84b)

### CODE REVIEW

**Comment by StanZGenchev** [comment]
I see unneeded setTimeout functions and the 1 xsaccess file per folder does not work properly.
Will discuss later.

**Comment by delchev** [comment]
- there are more file types where there is no name in it not only .xsaccess (e.g. .xspriviledges, etc.)
- these files like .xsaccess should not be processed here in this way, because they are not known in this context

File: ide-workspace/workspace.js [hunk 1]
```diff
@@ -1695,7 +1675,7 @@ angular.module('workspace', ['workspace.config', 'ideUiCore', 'ngAnimate', 'ngSa
     }]);
 
 const images = ['png', 'jpg', 'jpeg', 'gif'];
-const models = ['extension', 'extensionpoint', 'edm', 'model', 'dsm', 'schema', 'bpmn', 'job', 'xsjob', 'listener', 'websocket', 'roles', 'constraints', 'table', 'view'];
+const models = ['extension', 'extensionpoint', 'edm', 'model', 'dsm', 'schema', 'bpmn', 'job', 'xsjob', 'xsaccess', 'listener', 'websocket', 'roles', 'constraints', 'table', 'view'];
```
Comment (delchev) [inline ide-workspace/workspace.js]: xsjob and xsaccess are not known in this context
have to be done via extensions

**Review by delchev — CHANGES_REQUESTED** [review]
there are more file types where there is no name in it not only .xsaccess (e.g. .xspriviledges, etc.)
these files like .xsaccess should not be processed here in this way, because they are not known in this context

**Review by delchev — APPROVED** [review]

**Review by ThuF — APPROVED** [review]

**Pull request merged**